### PR TITLE
java-ocsp-stapling

### PR DESCRIPTION
**Pull request by scottf**

### CODE REVIEW

File: content/blog/java-ocsp-stapling.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-OCSP Stapling has been supported in Java since JDK 8. There is an excellent document from Oracle which describes it in detail.
+## OCSP Stapling SSLContext
 
-[Client-Driven OCSP and OCSP Stapling](https://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/ocsp.html)
+OCSP Stapling has been supported in Java since JDK 8. This excellent document from Oracle, [Client-Driven OCSP and OCSP Stapling](https://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/ocsp.html), describes it in detail.
 
 ### Examples
 
-Example of creating the SSLContext can be found in the [OCSP Example Class](src/main/java/io/nats/ocsp/OcspExample.java)
+Example of creating the `SSLContext` can be found in the [OCSP Example Class](src/main/java/io/nats/ocsp/OcspExample.java)
```
Comment (gcolliso) [inline content/blog/java-ocsp-stapling.md:22]: what is the full URL for the OCSP Example Class?

Comment (scottf) [reply content/blog/java-ocsp-stapling.md:22]: https://github.com/nats-io/java-nats-examples/blob/main/ocsp/src/main/java/io/nats/ocsp/OcspExample.java

Comment (scottf) [reply content/blog/java-ocsp-stapling.md]: fixed

**Review by scottf — COMMENTED** [review]
LGTM

**Comment by gcolliso** [comment]
lgtm

**Pull request merged**